apps/travel-showcase/src/app/styles.css

```
@@ -0,0 +1,368 @@
+  --ink-faint: #5d7365;
+  --page: #f4f7f5;
+  --surface: #ffffff;
+  --line: #d8e3dc;
+  --accent: #0f6b47;
+  --accent-ink: #ffffff;
+  --warn: #8a4a12;
+  --warn-bg: #fdf2e4;
+  --over: #a3231f;
+  --radius: 14px;
+  --gap: 20px;
+  color-scheme: light;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)